src/main/java/Duke.java

```diff
@@ -1,10 +1,107 @@
+// enums not applicable
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+        ArrayList<Task> arrlst = new ArrayList<>();
+        String DASH = "____________________________________________________________";
+        Action act = new Action();
+
+        act.greet();
+
+        while (sc.hasNext()) {
+            String phrase = sc.nextLine();
+            System.out.println(DASH);
+
+            if (phrase.equals("list")) {
```

Comment:
Perhaps you can consider using a switch statement instead?

Reply:
The code here is pretty long, but you will get to break it down later as you progress with the iP. Consider doing the printing in separate functions and get your switch statement to invoke those print functions instead.